src/components/figure/figure.scss

```diff
@@ -0,0 +1,34 @@
+@use "../../design-tokens/colors.yml";
+@use "../../mixins/ms";
+
+/**
+ * Removes default margin from <figure>
+ */
+.c-figure {
+  margin: 0;
+}
+
+/**
+ * Contain and center image
+ */
+.c-figure__object {
+  display: block;
```

Comment:
We are already setting `display: block` by default on media elements in our `base/_defaults` stylesheet.